pages/ugm2024.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title: UGM 2024
+---
+
+## When and Where
+The UGM will be held in-person across 3 days, August 21-23, 2024, in London, UK at [King's College London](https://www.kcl.ac.uk/visit/bush-house).
+
+## What to Expect
+The UGM will bring together users of the MDAnalysis package from different communities. Our goal is to foster an opportunity to connect for interdisciplinary researchers and developers across biomolecular simulations, soft matter, biophysics and more. Whether you are a new MDAnalysis contributor, an MDAnalysis core developer, or a researcher using (or interested in using) MDAnalysis for your work in academia or industry, this UGM is for you. You can expect:
```

Comment:
We only need "bio" once. Instead also mention materials. (I hope @micaela-matta agrees.)
```suggestion
The UGM will bring together users of the MDAnalysis package from different communities. Our goal is to foster an opportunity to connect for interdisciplinary researchers and developers across biomolecular simulations, soft matter, materials science and more. Whether you are a new MDAnalysis contributor, an MDAnalysis core developer, or a researcher using (or interested in using) MDAnalysis for your work in academia or industry, this UGM is for you. You can expect:
```